app/assets/stylesheets/govuk_publishing_components/components/_search.scss

```diff
@@ -42,20 +44,20 @@ $large-input-size: 50px;
 @if ($is-ie == false) or ($ie-version >= 8) {
```

Comment:
this i believe is still from govuk_frontend_toolkit

Reply:
Good spot! Will look into it

Reply:
Updated and removed these conditions which from my understanding were aimed to prevent some styles from being passed to IE6 and 7. The reasons for removing them is that:
- this library relies on govuk-frontend, which supports Internet Explorer down to version 8
- removing the conditions (adding these styles to older browsers) doesn't seem to affect the way the component is being displayed in IE6 and 7.

Reply:
👍